lib/view.coffee

```diff
@@ -250,7 +280,20 @@ class TerminusView extends View
     @animating = true
     @xterm.height if @maximized then @maxHeight else @prevHeight
 
+  openInDock: =>
+    @statusBar.setActiveTerminalView this
+    @statusIcon.activate()
+    atom.workspace.open(this, activatePane: true)
+    .then =>
+      if not @opened
+        @opened = true
+        @displayTerminal()
+        @xterm.height("100%")
+        @emit "terminusl:terminal-open"
```

Comment:
```suggestion
        @emit "terminus:terminal-open"
```

Reply:
Please check this box on sidebar

![Capture](https://user-images.githubusercontent.com/31389848/68762372-572f8280-0616-11ea-9de4-9eddb3bf99e8.PNG)